students/tfbanks/Lesson03/Mailroom_pt1.py

```diff
@@ -0,0 +1,120 @@
+# mailroom_pt1.py Exercise by tfbanks
+
+# !/usr/bin/env python3
+
+# Donor Database containing names and donations
+donor_db = [
+    ['Tim Cooker', 1500.50, 25050.50, 15680.75],
+    ['Elon Musket', 5250.25, 26500],
+    ['Frank Petersmankempt', 550.60],
+    ['Megan Morgan', 650.40, 1600, 20625.40],
+    ['Marlene Wheeler', 820, 1222.80]
+]
+
+donor = []  # Craate Donor List
+for dname in donor_db:
+    d = dname[0]
+    donor.append(d)
+
+
+def thank_you():  # Code for selecting Donor and writing a Thank You Note
+    ty_donor = input('Enter the Full Name of a donor (for list of previous donors type list): ')
+
+    if ty_donor.title() == 'List':  # If List, Prints Donor List and restarts the function
+        print(donor)
+        thank_you()
+    elif ty_donor.title() in donor:   #  If Donor is on list, print a letter thanking them for another donation
+        d_name = donor.index(ty_donor.title())
```

Comment:
this var does not represent donor name, it actually represents `donor_index` or `donor_db_index` if you want to be really verbose.

remember that good var names help the reader that reads your code.. and code is more often read than written 🙂